src/SDKs/Azure.Base/data-plane/Azure.Base/Http/Pipeline/ExponentialRetryPolicy.cs

```diff
@@ -0,0 +1,60 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Linq;
+
+namespace Azure.Base.Http.Pipeline
+{
+    internal class ExponentialRetryPolicy : RetryPolicy
```

Comment:
How do I create an instance?

Reply:
I can make it public here but was going to do it as part of https://github.com/Azure/azure-sdk-for-net/issues/5632.